packages/excalidraw/actions/actionFrame.ts

```diff
@@ -144,3 +146,46 @@ export const actionSetFrameAsActiveTool = register({
     !event.altKey &&
     event.key.toLocaleLowerCase() === KEYS.F,
 });
+
+export const actionWrapSelectionInFrame = register({
+  name: "wrapSelectionInFrame",
+  label: "labels.wrapSelectionInFrame",
+  trackEvent: { category: "element" },
+  predicate: (elements, appState, _, app) => {
+    const selectedElements = getSelectedElements(elements, appState);
+
+    return (
+      selectedElements.length > 0 &&
+      !selectedElements.some((element) => isFrameLikeElement(element))
+    );
```

Comment:
this predicate allows adding subgroups into frame, where only part of the group is assigned to the frame and the rest is not, which is technically not allowed.

But the same issue is present in production when creating a new frame over a partial group, so we'll fix that separately.